From 666fd7f69f684d7dd23b1ee665b0902e29953b76 Mon Sep 17 00:00:00 2001
From: graysky <graysky@archlinux.us>
Date: Sat, 12 Dec 2020 09:29:51 -0500
Subject: [PATCH] mcpu=cortex-a72 for RPi4 application to ffmpeg

Get build system to patch ffmpeg source removing the -march= flag

---
 cmake/modules/FindFFMPEG.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/modules/FindFFMPEG.cmake b/cmake/modules/FindFFMPEG.cmake
index 9ce807c..a18b21c 100644
--- a/cmake/modules/FindFFMPEG.cmake
+++ b/cmake/modules/FindFFMPEG.cmake
@@ -279,6 +279,7 @@ if(NOT FFMPEG_FOUND)
                                     ${CMAKE_SOURCE_DIR}/tools/depends/target/ffmpeg/FindGnuTls.cmake
                                     <SOURCE_DIR> &&
                                     patch -p1 < ${CMAKE_SOURCE_DIR}/tools/depends/target/ffmpeg/0001-rpi-Add-hevc-acceleration.patch &&
+                                    patch -p1 < ${CMAKE_SOURCE_DIR}/tools/depends/target/ffmpeg/0002-use-mcpu-avoiding-march-and-mtune.patch &&
                                     echo "########################################## patched ffmpeg ##############################"
                      )
 
-- 
2.29.2

